src/reducers/field006.spec.js

```diff
@@ -0,0 +1,28 @@
+import chai from 'chai';
+import fs from 'fs';
+import path from 'path';
+import {MarcRecord} from '@natlibfi/marc-record';
+import createReducer from './field006';
+import fixturesFactory, {READERS} from '@natlibfi/fixura';
+import createDebugLogger from 'debug';
+
+MarcRecord.setValidationOptions({subfieldValues: false});
+
+describe('reducers/field006', () => {
+  const debug = createDebugLogger('@natlibfi/melinda-marc-record-merge-reducers');
+  const {expect} = chai;
+  const fixturesPath = path.join(__dirname, '..', '..', 'test-fixtures', 'reducers', 'field006');
+
+  fs.readdirSync(fixturesPath).forEach(subDir => {
+    const {getFixture} = fixturesFactory({root: [fixturesPath, subDir], reader: READERS.JSON, failWhenNotFound: false});
+    it(subDir, () => {
+      const base = new MarcRecord(getFixture('base.json'));
+      const source = new MarcRecord(getFixture('source.json'));
+      const tagPattern = new RegExp(getFixture({components: ['tagPattern.txt'], reader: READERS.TEXT}), 'u');
```

Comment:
Since tag pattern stays as same for whole test and is one liner, it can be hard coded.